tests/e2e/ansible/install_build_deps.yml

```diff
@@ -13,10 +13,16 @@
         name: "{{ build_pkgs[ansible_distribution | lower] }}"
         state: present
     - block:
-      - name: Download and extract Go tarball
+      - name: Download Go tarball
```

Comment:
This answers one of my questions on the previous commit :D

Reply:
Perhaps run `get_url` with `force: true` to update the tgz in case it exists and the content changed.

Reply:
On the other hand the workers should be always clean and locally we might benefit of the caching, what do you think?

Reply:
Caching is nice. Then rename `go.tar.gz` -> `go{{ go_version }}.linux-{{ target_arch }}.tar.gz` so that a version update will force the download?

Reply:
yep, that sounds good